lib/resources/r4/procedure.yaml

```diff
@@ -49,3 +52,204 @@ fields:
+    {
+      "location": {
+        "reference":"Location/1693915",
+        "display": "Baseline East"
+      }
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.location
+
+- name: performed_x_
+  required: 'No'
+  cardinality: 0..1
+  type: dateTime
+  description: Estimated or actual date or date-time when the procedure was performed.
+  example: |
+    {
+      "performedDateTime": "2015-10-14T13:13:20-06:00"
+    }
+  url: http://hl7.org/fhir/procedure-definitions.html#Procedure.performed_x_
+  note: |
+    <ul>
+      <li> Only a type of dateTime is supported.</li>
+      <li> The dateTime is converted to a GMT date. </li>
+    </ul>
+
+- name: performer
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  description: Limited to "real" people rather than equipment.
+  example: |
+    {
+      "performer": [
+        {
+          "function": {
+            "coding": [
+              {
+                "system": "http://snomed.info/sct",
+                "code": "223366009",
+                "display": "Healthcare professional (occupation)",
+                "userSelected": true
+              }
+            ],
+            "text": "Healthcare professional (occupation)"
+          },
+          "actor": {
+            "reference": "Practitioner/2150097",
+            "display": "Strecker, Bradley"
+          }
+        }
+      ]
+    }
+  url: https://hl7.org/fhir/procedure-definitions.html#Procedure.performer
+  children:
+
+  - name: function
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description: Distinguishes the type of involvement of the performer in the procedure.
+    example: |
+      {
+        "function": {
+          "coding": [
+            {
+              "system": "http://snomed.info/sct",
+              "code": "223366009",
+              "display": "Healthcare professional (occupation)",
+              "userSelected": true
+            }
+          ],
+          "text": "Healthcare professional (occupation)"
+        }
+      }
+    example2: |
+      {
+        "severity": {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/388",
+              "code": "1207",
+              "display": "Anesthesiologist",
+              "userSelected": true
+            }
+          ],
+          "text": "Anesthesiologist"
+        }
+      }
+    url: https://hl7.org/fhir/procedure-definitions.html#Procedure.performer.function
+
+  - name: actor
+    required: 'Yes'
+    cardinality: 1..1
+    type: Reference
+    description: The practitioner who was involved in the procedure.
+    example: |
+      {
+        "actor": {
+          "reference": "Practitioner/2150097",
+          "display": "Strecker, Bradley"
+        }
+      }
+    url: http://hl7.org/fhir/procedure-definitions.html#Procedure.performer.actor
+    note: performer.actor must be a reference to a Practitioner.
```

Comment:
Must only include references to Practitioner.

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/372/commits/edd713404a0347648bfd408e254be8a2d11466ce#diff-80d4a0c17acadba0cd26a39c2cfeaf8dR243).